features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2155,4 +2155,16 @@
 
     <!-- Configuration for allowing an uninterrupted token building flow upon facing claim handling errors. -->
     <ContinueOnClaimHandlingError>{{continue_on_claim_handling_error}}</ContinueOnClaimHandlingError>
+
+    <!-- CORS -->
+    <CORS>
+        <AllowGenericHttpRequests>true</AllowGenericHttpRequests>
+        <AllowSubdomains>false</AllowSubdomains>
+        <SupportedMethods>GET, POST, HEAD, OPTIONS</SupportedMethods>
+        <SupportedHeaders>*</SupportedHeaders>
+        <ExposedHeaders></ExposedHeaders>
+        <SupportsCredentials>true</SupportsCredentials>
+        <MaxAge>-1</MaxAge>
+        <TagRequests>false</TagRequests>
+    </CORS>
```

Comment:
shouldn't we allow these values to be configurable via the deployment.toml?

Reply:
I'll change this.